src/components/shared/SocialIconLinkComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import type { SocialIconLinkProps } from '@/interfaces/SocialIconLinkProps.ts';
+
+export const SocialIconLinkComponent: React.FC<SocialIconLinkProps> = ({ children, url, customClass, target }) => {
+    const linkProps: any = { rel: 'noreferrer' };
```

Comment:
```suggestion
    const linkProps: HTMLAnchorElement = { rel: 'noreferrer' };
```
BTW https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement

Reply:
Linter returns "type {type: string} is not assignable to variable type HTMLAnchorElement" error. So I've created `LinkProps` interface